src/bluehawk/parser/parser.test.ts

```diff
@@ -426,7 +426,18 @@ this is not bluehawk markup
       expect(parser.errors.length).toBe(0);
     });
 
-    it("rejects block commands that straddle comment blocks", () => {
+    it("accepts block commands that straddle inside of comment blocks", () => {
+      const test_string = `:block-command-start: */
+chunky chunky
+/* :block-command-end:`
+      const result = lexer.tokenize(test_string);
+      //expect(result.errors.length).toBe(0);
+      parser.input = result.tokens;
+      parser.blockCommand();
+      expect(parser.errors).toStrictEqual([]);
+    });
+
+    it("rejects block commands that straddle outside of comment blocks", () => {
```

Comment:
Please add some tests:

- multiple block comment ends should produce error
- block comment end not allowed within attribute list